docs/document/content/manual/sharding-jdbc/configuration/config-java.cn.md

```diff
@@ -285,15 +285,22 @@ ShardingStrategyConfiguration的实现类，用于配置不分片的策略。
 
 #### KeyGeneratorConfiguration
 
-| *名称*             | *数据类型*                    | *说明*                                                                         |
-| ----------------- | ---------------------------- | ------------------------------------------------------------------------------ |
-| column            | String                       | 自增列名称                                                                      |
-| type              | String                       | 自增列值生成器类型，可自定义或选择内置类型：SNOWFLAKE/UUID/LEAF_SEGMENT/LEAF_SNOWFLAKE|
-| props             | Properties                   | 自增列值生成器的相关属性配置                                                      |  
+| *名称*                 | *数据类型*                    | *说明*                                                                         |
+| --------------------- | ---------------------------- | ------------------------------------------------------------------------------ |
+| column                | String                       | 自增列名称                                                                      |
+| keyGenerateAlgorithm  | KeyGenerateAlgorithm         | 自增列值生成算法                                                                 |
+
+#### KeyGenerateAlgorithm 
```

Comment:
explain of KeyGenerateAlgorithm could be removed

Reply:
done